packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/object/object/addUser/index.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+import '@ovh-ux/manager-core';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import 'ovh-api-services';
+
+import containers from '../../../../../containers';
+import routing from './addUser.routing';
+
+const moduleName = 'ovhManagerPciStoragesObjectsObjectsObjectAddUser';
```

Comment:
Hard to understand the difference between `ObjectsObjects` and `ObjectsObjectsObject `

Reply:
Yes i agree, ill take care of this once these small PRs are merged. Cannot work on this until i have all of it in 1 PR, once 
https://github.com/ovh/manager/pull/5373
https://github.com/ovh/manager/pull/5369
https://github.com/ovh/manager/pull/5319

are merged to `feat/add-s3-users`, i can change the states of all modules at once